configure.ac

```diff
@@ -305,6 +305,11 @@ AC_CHECK_HEADER(errno.h,
                 HAVE_ERRNO_H=1
                 AC_DEFINE(HAVE_ERRNO_H, 1),
                )
+AC_SUBST(HAVE_SYS_CDEFS_H)
+AC_CHECK_HEADER(sys/cdefs.h,
+                HAVE_SYS_CDEFS_H=1
+                AC_DEFINE(HAVE_SYS_CDEFS_H, 1),
+               )
 AC_SUBST(HAVE_SYS_STAT_H)
```

Comment:
Btw, I don't see why you do all this when a simple AC_CHECK_HEADER() does AC_SUBST for you, but maybe that's just legacy.

Reply:
If was the first major configure script I made, and I used configure script from other projects as a template. I can do some testing and probably remove some of the AC_SUBST

Reply:
Yeah unless you use the HAVE_* directly in configure.ac you should only need `AC_CHECK_HEADER(foo.h)` each time.
If you need to use the result in configure.ac you can do so with `$ac_cv_HAVE_FOO_H` IIRC.